cmd/tailscaled/childproc/childproc.go

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2022 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package childproc allows other packages to register be-child process
```

Comment:
Could you add a reference to the top-level command you mean? It took me a later function docstring to realize that "be-child" meant specifically "a subcommand of tailscaled"

Reply:
read the next line? it says "with tailscaled" ?

Reply:
reworded